### ISSUE TITLE
Admin account not created

### ISSUE THREAD

**Issue by phretor** [tiredofit/docker-freescout#20]
Is it just me or the admin account doesn't get created?

**Comment by DmitroChemniy** [comment]
Hi! I have the same problem
<img width="1432" alt="Screenshot 2020-05-22 at 14 00 22" src="https://user-images.githubusercontent.com/8407318/82662266-39328280-9c36-11ea-848b-bcb395aa8fda.png">

**Comment by tiredofit** [comment]
This is the command we are using to create:

`sudo -u ${NGINX_USER} php artisan freescout:create-user --role=admin --firstName=${ADMIN_FIRST_NAME} --lastName=${ADMIN_LAST_NAME} --email=${ADMIN_EMAIL} --password=${ADMIN_PASS}`  

Can you try that out inside the app container and see if that resolves your issues? Might need to be run inside /www/freescout

**Comment by DmitroChemniy** [comment]
so it helped, but immediately after authorization there is a redirect to localhost:port

**Comment by tiredofit** [comment]
I'm not sure I follow. Can you provide some screenshots and steps that happened? I won't be able to fully look till mid next week as to whats happening but any more info will help and potentially get a fix out sooner.

**Comment by DmitroChemniy** [comment]
on the frontend there is nginx
server {
	listen 80;
	server_name my-domain.com;

        access_log  /var/log/nginx/freescout_access.log;
        error_log   /var/log/nginx/freescout_error.log;

	location / {
		proxy_pass http://localhost:9302/;
	}
}

after entering the login / password on the authorization page
http://my-domain.com/login

in the system authorizes and redirects to url localhost:9302

in order to get to the interface you need to go to my-domain.com again and then everything is ok

**Comment by Leopere** [comment]
I'm unable to log in with the `docker-compose.yml` defined 
```yaml
    - ADMIN_EMAIL=USER@REDACTED.com
    - ADMIN_PASS=REDACTED
```

**Comment by DmitroChemniy** [comment]
> I'm unable to log in with the `docker-compose.yml` defined
> 
> ```yaml
>     - ADMIN_EMAIL=USER@REDACTED.com
>     - ADMIN_PASS=REDACTED
> ```

The decision was higher [https://github.com/tiredofit/docker-freescout/issues/20#issuecomment-632663247](https://github.com/tiredofit/docker-freescout/issues/20#issuecomment-632663247)

**Comment by Leopere** [comment]
root@t1:~|⇒  docker exec -ti freescout_freescout-app.1.0d80ezm7y15j2bbnowlimvrpr /bin/bash
bash-5.0# freescout_freescout-app.1.0d80ezm7y15j2bbnowlimvrpr^C
bash-5.0# sudo -u ${NGINX_USER} php artisan freescout:create-user --role=admin --firstName=${ADMIN_FIRST_NAME} --lastName=${ADMIN_LAST_NAME} --email=${ADMIN_EMAIL} --password=${ADMIN_PASS}
Could not open input file: artisan
bash-5.0#

**Comment by Leopere** [comment]
```bash
cd /www/html && sudo -u ${NGINX_USER} php artisan freescout:create-user --role=admin --firstName=${ADMIN_FIRST_NAME} --lastName=${ADMIN_LAST_NAME} --email=${ADMIN_EMAIL} --password=${ADMIN_PASS}
```
Here is a revised command for you.

**Comment by DmitroChemniy** [comment]
> ```shell
> cd /www/html && sudo -u ${NGINX_USER} php artisan freescout:create-user --role=admin --firstName=${ADMIN_FIRST_NAME} --lastName=${ADMIN_LAST_NAME} --email=${ADMIN_EMAIL} --password=${ADMIN_PASS}
> ```
> 
> Here is a revised command for you.

Yep. Might need to be run inside /www/freescout

**Comment by Leopere** [comment]
Yeah I just updated the command to be a oneliner instead of the side instruction and it's not `/www/freescout` unfortunately its `/www/html` to be more precise for future netizens.

**Comment by jonathanvansina** [comment]
> ```shell
> cd /www/html && sudo -u ${NGINX_USER} php artisan freescout:create-user --role=admin --firstName=${ADMIN_FIRST_NAME} --lastName=${ADMIN_LAST_NAME} --email=${ADMIN_EMAIL} --password=${ADMIN_PASS}
> ```
> 
> Here is a revised command for you.

Same here.. The oneliner works fine thanks @Leopere..
Still looking for a fix. Logs are showing this:
```bash
freescout-db           | 2020-06-19 20:34:47 6 [Warning] 'user' entry 'root@e2e94d64c09a' ignored in --skip-name-resolve mode.
freescout-db           | 2020-06-19 20:34:47 6 [Warning] 'user' entry '@e2e94d64c09a' ignored in --skip-name-resolve mode.
freescout-db           | 2020-06-19 20:34:47 6 [Warning] 'proxies_priv' entry '@% root@e2e94d64c09a' ignored in --skip-name-resolve mode.
freescout-db           | 2020-06-19 20:34:47 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
freescout-db           | 2020-06-19 20:34:47 0 [Note] InnoDB: Buffer pool(s) load completed at 200619 20:34:47
freescout-db           | 2020-06-19 20:34:47 6 [Warning] 'user' entry 'root@e2e94d64c09a' ignored in --skip-name-resolve mode.
freescout-db           | 2020-06-19 20:34:47 6 [Warning] 'user' entry '@e2e94d64c09a' ignored in --skip-name-resolve mode.
freescout-db           | 2020-06-19 20:34:47 6 [Warning] 'proxies_priv' entry '@% root@e2e94d64c09a' ignored in --skip-name-resolve mode.

```

**Comment by jonathanvansina** [comment]
This problem is solved in #24. This ticket can be closed.
